guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -197,6 +256,20 @@ If `eth0` was renamed to `em0`, call the installer to use the new interface name
 # leapp upgrade
 ----
 
+ifdef::satellite[]
++
+If you run {Project} in a disconnected environment, add the following parameters:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# leapp preupgrade \
```

Comment:
why is it `leapp preupgrade` twice? @evgeni made a comment on consolidating these two sections.

Reply:
Removed.